frontend/src/metabase/static-viz/components/XYChart/Values/Values.tsx

```diff
@@ -150,10 +150,10 @@ export default function Values({
               {shouldRenderDataPoint && (
                 <circle
                   key={index}
-                  r={2}
+                  r={3}
```

Comment:
Make data point circles easier to see. And looks similar size as the app viz data point circles.

#### After
![image](https://user-images.githubusercontent.com/1937582/189328157-6ce634d6-5687-45a7-8d63-aa456873e6f5.png)

#### Before
![image](https://user-images.githubusercontent.com/1937582/189328204-8720945c-ce87-479c-b555-4d498ab39761.png)